config/environments/production.rb

```diff
@@ -87,4 +94,14 @@
 
   # Use Let's Encrypt for SSL
   config.middleware.use Letsencrypt::Middleware
+
+  # Use a different logger for distributed setups.
+  # require 'syslog/logger'
+  # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
+
+  if ENV["RAILS_LOG_TO_STDOUT"].present?
```

Comment:
Looks like they worked `rails_12factor` in? Might be okay to remove it now, I'll investigate